MozillaFoundation/foundation.mozilla.org
# Update `add item` widget button margins in Wagtail edit page UI.

# Description

This PR attempts to fix the misleading white-space that exists around the `.c-sf-add-button` elements, by displaying them closer to the previous element rather than the next one.

Current state:
![image](https://github.com/user-attachments/assets/68efc620-5bbb-43b8-9d2c-48a2f29f00c9)

Fixed state:
<img width="966" alt="image" src="https://github.com/user-attachments/assets/9492d112-4ab9-4955-b239-485c0c234804">

# To test

1. Login to the cms in the [preview app](https://foundation-s-tp1-561-ma-nwd1qy.herokuapp.com/cms/login/).
2. Authenticate with the credentials: `admin2` for both user and password.
3. Open any page for editing. Association by proximity should be elevated in the `Add item` buttons of the block widgets, it should reflect that it will append an item right after the element above.

Link to sample test page: https://foundation-s-tp1-561-ma-nwd1qy.herokuapp.com/cms/login/
Related PRs/issues: #12247

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-939)
